controllers/remove_process_groups.go

```diff
@@ -408,18 +409,15 @@ func (r *FoundationDBClusterReconciler) removeProcessGroups(ctx context.Context,
 	// We have to check if the currently removed process groups are completely removed.
 	// In addition, we have to check if one of the terminating process groups has been cleaned up.
 	for _, processGroup := range processGroups {
-		removed, include, err := confirmRemoval(ctx, logger, r, cluster, processGroup)
-		if err != nil {
+		include, err := confirmRemoval(ctx, logger, r, cluster, processGroup)
+		if err != nil && !internal.IsResourceNotDeleted(err) {
```

Comment:
shouldn't this be an or? if there is a non-resource-deleted error it won't log it (though need to ensure it's happy with a nil error in errors.As())

Reply:
ohh I see, yup you are correct